Project.toml

```diff
@@ -28,6 +28,7 @@ PowerModels = "c36e90e8-916a-50a6-bd94-075b64ef4655"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Quadmath = "be4d8f0f-7fa4-5f49-b795-2f01399ab2dd"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SCS = "c946c3f1-0d1f-5ce8-9dea-7daa1f7e2d13"
```

Comment:
I would rather we do not add extra dependencies if we can avoid it.
The SDP formulation is numerically and computationally tough, users should probably use Mosek here anyway.